source/common/config/tls_context_json.cc

```diff
@@ -61,6 +61,11 @@ void TlsContextJson::translateCommonTlsContext(
     common_tls_context.mutable_tls_params()->add_cipher_suites(std::string{cipher_suite});
   }
 
+  // TODO: this doesn't currently support inline data
```

Comment:
We won't ever support inline in v1. I would remove this TODO. Did you add docs for v1 also in data-plane-api? I can't remember.

Reply:
Removed!  And yes, I did add documentation [here][docs].

[docs]: https://github.com/envoyproxy/data-plane-api/blob/master/docs/root/api-v1/listeners/listeners.rst#user-content-config-listener-ssl-context-crl-file